purchase_open_qty_stock_uom/models/purchase_order_line.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2025 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class PurchaseOrderLine(models.Model):
+    _inherit = "purchase.order.line"
+
+    @api.depends("qty_to_receive", "product_uom", "product_id.uom_id")
+    def _compute_qty_to_receive_stock_uom(self):
```

Comment:
We might as well extend the existing method.

```suggestion
    def _compute_qty_to_receive(self):
        super()._compute_qty_to_receive()
```